test/run-spec-wasm2c.py

```diff
@@ -145,7 +156,10 @@ def GetModulePrefix(self, idx_or_name=None):
     def GetModulePrefixUnmangled(self, idx):
         return self.unmangled_names[idx]
 
-    def _CacheModulePrefixes(self):
+    def GetModuleInstanceName(self, idx_or_name=None):
+        return self.GetModulePrefix() + '_module_instance'
+
+    def CacheModulePrefixes(self):
```

Comment:
Why not keep the leading `_` on this method name?

Reply:
We wanted it to be "public" so we can call it within the main function shortly after creating the CWriter (and before cwriter.GetModulePrefixUnmangled() gets called on every module).

Reply:
Fair enough; latest commit keeps the leading `_` (which I think means "private"?) and adds a call from within the CWriter's `__init__`. (We need this to be called before GetModulePrefixUnmangled but we need wasm2c to run before cwriter.Write(), so it can parse the wasm2c-generated `.h` files.)